### ISSUE TITLE
Fix accessibility issues in supported styles/widgets

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#731]
These are problems found via pa11y in https://github.com/unl/wdntemplates/pull/730

These are important to fix because these problems are causing sites to fail in http://webaudit.unl.edu

To find out more information about these problem, go to the examples page for each test and run the SiteMaster Pa11y bookmarklet.

For example, to find out more about jqueryui.html, go to http://wdn.unl.edu/wdn/templates_4.0/examples/#jqueryui.html

```
jqueryui.html FAILED!
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
    WCAG2AA.Principle1.Guideline1_3.1_3_1.H44.2
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Value
    WCAG2AA.Principle1.Guideline1_3.1_3_1.H44.2

table.html FAILED!
    WCAG2AA.Principle1.Guideline1_3.1_3_1.H43.IncorrectAttr
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

buttons.html FAILED!
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

videoplayer.html FAILED!
    WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2

colorbox.html FAILED!
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
    WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name

zenboxes.html FAILED!
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail

tabs.html FAILED!
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
    WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
```
